tests/blocks/test_core.py

```diff
@@ -2477,3 +2477,30 @@ class Baz_Alias(Block):
 
         # new local schema ID should now be saved to Prefect
         assert updated_schema_id == new_schema_id
+
+
+class TestDeleteBlock:
+    @pytest.fixture
+    def NewBlock(self):
+        # Ignore warning caused by matching key in registry due to block fixture
+        warnings.filterwarnings("ignore", category=UserWarning)
+
+        class NewBlock(Block):
+            a: str
+            b: str
+
+        return NewBlock
+
+    async def test_delete_block(self, NewBlock):
+        new_block = NewBlock(a="foo", b="bar")
+        new_block_name = "my-block"
+        await new_block.save(new_block_name)
+
```

Comment:
```suggestion
        await new_block.save(new_block_name)
        await new_block.load(new_block_name)

```

Not terribly important, just a stylistic thing.  In tests, I often like to show that something would work before the operation I'm testing, then show how it works after my operation.  So in this case, I can `.load` it before, then I `.delete` it, so afterward I can't `.load` it again.  Your test is totally fine without it, just a style thing.

Reply:
Thanks for the input @chrisguidry , and completely agreed that it makes the intent of the test much clearer. Added the same now.